boards/arm/stm32f3_seco_d23/doc/index.rst

```diff
@@ -0,0 +1,227 @@
+.. _stm32f3_seco_d23_board:
+
+SECO JUNO SBC-D23
+####################
+
+Overview
+********
+
+JUNO (SBC-D23) is a Single Board Computer based on embedded Rockchip PX30
+Processor, featuring Quad-Core ARM Cortex-A35 processor. The processor
+integrates a Mali-G31 GPU with High performance dedicated 2D processor,
+supporting OpenGL ES 1.1 / 2.0 / 3.2, Vulkan 1.0, OpenCL 2.0 and Open VG 1.1.
+Embedded VPU is able to support video decoding of the most common coding
+standard (MPEG-4, H.265/HEVC, H.264, VP8, VC-1). The board is completed with up
+to 4GB LPDDR4-3200 32-bit bus memory directly soldered on board and one eMMC
+5.1 Flash Drive with up to 64GB of capacity. LVDS Single Channel interface and
+HDMI are supported. The RMII interface and Micrel KSZ8091 Ethernet Transceiver
+allow the implementation of a Fast Ethernet interface. The networking
+capabilities can be extended by WiFi+BT M.2 module and external modem module.
+The audio functionalities are managed by the AudioCodec embedded in the RK-809
+PMIC. The JUNO board is completed by a series of connectors with various
+interfaces (UART, SPI, I2C) managed by the microcontroller STM32F302VCT6.
+
+.. image:: img/stm32f3_seco_d23.jpg
+     :width: 350px
+     :align: center
+     :height: 531px
```

Comment:
nit: avoid setting manual width/height, just provide a board picture that has the right size (current picture has black background, should be removed)

Reply:
Fixing and converting to png